src/main/java/Duke.java

```diff
@@ -1,10 +1,62 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    public static void printList(List<Task> l1) {
+        int index;
+        for (int i = 0; i < l1.size(); i += 1) {
+            index = i + 1;
+            System.out.println(index + ". [" + l1.get(i).getStatusIcon() + "] " + l1.get(i).getDescription());
+        }
+        System.out.println();
+    }
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+        /*String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello from\n" + logo);*/
+        System.out.println("Hello! I'm a Robot");
+        System.out.println("What can I do for you?\n");
+        List<Task> list = new ArrayList<>();
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+        while(!line.equals("bye")) {
+            if (line.equals("list")) {
+                if (!list.isEmpty()) {
+                    System.out.println();
+                    printList(list);
+                } else {
+                    System.out.println('\n' + "List is empty!" + '\n');
+                }
+            } else if (line.startsWith("mark")) {
+                int list_index = Integer.parseInt(line.substring(5)) - 1;
+                if (list.get(list_index).getIsDone()) {
```

Comment:
I like how you have added an extra if statement to check if the task is marked/unmarked. This reduces the task for your program to mark/unmark it again which is quite efficient.